pkgs/tools/misc/coreutils/default.nix

```
@@ -60,12 +69,12 @@ stdenv.mkDerivation (rec {
 
     # intermittent failures on builders, unknown reason
     sed '2i echo Skipping du basic test && exit 77' -i ./tests/du/basic.sh
-  '' + (optionalString (stdenv.hostPlatform.libc == "musl") (lib.concatStringsSep "\n" [
+  '' + (lib.optionalString (stdenv.hostPlatform.libc == "musl") (lib.concatStringsSep "\n" [
     ''
       echo "int main() { return 77; }" > gnulib-tests/test-parse-datetime.c
       echo "int main() { return 77; }" > gnulib-tests/test-getlogin.c
     ''
-  ])) + (optionalString stdenv.isAarch64 ''
+  ])) + (lib.optionalString stdenv.isAarch64 ''
```

```suggestion
  ]) + lib.optionalString stdenv.isAarch64 ''
```
FYI these are optional.

---

I merely kept them.